src/app.ts

```diff
@@ -129,6 +129,18 @@ export class CeramicAnchorApp {
       )
       Metrics.count('HELLO', 1)
       logger.imp('Metrics exporter started')
+      if (this.config.metrics.instanceIdentifier) {
+         // In the API server, the instanceIdentifer is set from ECS_CONTAINER_METADATA_URI
```

Comment:
If done using the `entrypoint.sh` script above, we won't need to parse this in the code here.

Reply:
right!  the logic isn't quite the same - not sure if it matters, i was defaulting to the full uri if there wasn't a task id at the end - but i agree yours is better, since it moves it to the deployment and therefore we can make it very specific to what we expect!